front/SettingsForm.tsx

```diff
@@ -107,6 +111,7 @@ const SettingsForm = () => {
           <button
             className="btn btn-lg btn-primary pull-xs-right"
             type="submit"
+            data-cy="update-btn-settings"
```

Comment:
Consider using a more descriptive name for the data-cy attribute, such as 'update-settings-btn' instead of 'update-btn-settings'.